src/software/geom/algorithms/contains.cpp

```diff
@@ -14,7 +14,8 @@ bool contains(const Circle& container, const Segment& contained)
 
 bool contains(const Circle& container, const Point& contained)
 {
-    return distance(container.origin(), contained) <= container.radius();
+    return distanceSquared(container.origin(), contained) <=
```

Comment:
is this a bug fix?

Reply:
No this is just an optimization for the contains method